lib/plugins/aws/provider.js

```diff
@@ -393,6 +396,15 @@ class AwsProvider {
                     anyOf: [
                       { const: '*' },
                       { type: 'array', items: { $ref: '#/definitions/awsArn' } },
+                      {
```

Comment:
Looks great :+1: I think we can remove the line above as it's already covered by the definition you've added here

Reply:
good spot, missed that in my hurry to add mixed types

Reply:
no worries :+1: